### DISCUSSION TITLE
Mitmproxy is not recognizing some requests coming from a local Delphi system (newbie)

### DISCUSSION THREAD

**Discussion by lucas-Jose-Garcia** [mitmproxy/mitmproxy#6224] [Q&A]
Hello guys!
I'm new to mitmproxy and would like to use it to intercept some requests from the system that I'm automating, even though I've configured the certificate, some requests from that local system appear and others don't.

The only difference that I could identify is that these requests are being made using a specific component of Delphi, which would be Indy - Internet Direct and when they use the SOAP standard they are being identified.

Thank you all!

**Comment by mhils** [comment]
Did you take a look at the event log? :)

**Reply by lucas-Jose-Garcia** [reply]
Yes, I looked at the logs, but there really aren't any connection errors with these requests that are not being recognized.
I imagine it's something related to the system, because other proxy programs can't recognize these requests either.

**Reply by lucas-Jose-Garcia** [reply]
Just to complement, I tried to identify the request by Wireshark and it was possible to locate it, maybe it's some configuration that I'm not doing. I'll leave a print of Wireshark.
![Screenshot_12](https://github.com/mitmproxy/mitmproxy/assets/104788757/7f61cc62-9a2c-40af-85fc-2e00546120bb)
POST and GET requests to /Repom are not being acknowledged.

I also did a test of leaving a proxy activated without having any program managing that port and even so in these operations the system was able to receive a response.
So I imagine that this system is bypassing the proxy to make these requests.
